apps/web-connect/src/features/checkout/components/ChooseQuantityRow.tsx

```diff
@@ -59,8 +67,8 @@ export function ChooseQuantityRow(): JSX.Element {
                     <div className="flex md:gap-[21px] gap-[10px]">
                         <span className="block mt-2"><WarningIcon /></span>
                         <div>
-                            <span className="block font-bold text-lg">100 Maximum Per Transaction</span>
-                            <span className="block font-medium text-lg">A maximum of 100 NodeLicenses can be purchased in a single transaction.</span>
+                            <span className="block font-bold text-lg">175 Maximum Per Transaction</span>
```

Comment:
Please make this use the contant here too

Reply:
Done!